sem_2/hw_2_2.py

```diff
@@ -0,0 +1,17 @@
+# Напишите программу, которая получает целое число и возвращает его шестнадцатеричное строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+num = int(input("Введите целое число: "))
+print(hex(num))
+hex_map = {10: 'a', 11: 'b', 12: 'c', 13: 'd', 14: 'e', 15: 'f'}
```

Comment:
Можно добавить остальные ключи-цифры з этой системы или вообще использовать строку, при этом но остальной код будет работать также без изменений (смотрите ниже)
```suggestion
hex_map = '0123456789abcdef'
```

Reply:
Ну елы палы. Как же я сам не догадался)))